### ISSUE TITLE
Correction: TACL publication month

### ISSUE THREAD

**Issue by nschneid** [acl-org/acl-anthology#329]
https://aclweb.org/anthology/papers/Q/Q13/Q13-1019/ lists the date as December 2013, but in the footer of the PDF it says "Published 5/2013".

It looks like a systematic problem with TACL article months. E.g. https://aclweb.org/anthology/papers/Q/Q18/Q18-1001/ is listed as July 2018 but should be January 2018.

**Comment by mjpost** [comment]
Yes, this is a problem in the data that MIT Press hands us. These should be corrected manually until they fix their issues. CC: @aryamccarthy

**Comment by aryamccarthy** [comment]
Just opened a stopgap pull request. It's infeasible (or at least not a good use of time—duplicated effort) to download all of the PDFs, scrape them, and update the content when MIT Press is updating this information on their side.

In the meantime, if we don't have anything ~nice~ correct to say, let's not say anything at all. Month information for TACL was new; deleting it for now reverts us to the status quo ante.

**Comment by mjpost** [comment]
This was fixed in #409, right, @aryamccarthy?

**Comment by aryamccarthy** [comment]
Don’t get too excited too soon. Only the latest data have months; previous volumes’ data from the data source still universally say December.

-am
